### ISSUE TITLE
palles77 need your help

### ISSUE THREAD

**Issue by Valery813** [julius-speech/julius#149]
Dear palles77, you have trained a language model for Polish and English. Share with me the documentation or a sequence of actions on how to do it yourself. To make it clear to the novice. Halyards *.deduped is? what to do with them. I want to get a ready-made model for Julius or Kaldi.

**Comment by Valery813** [comment]
Maybe there is a training video ?

**Comment by palles77** [comment]
Hi Valery. I appreciate your enthusiasm with willing to learn about creation of language models. It is a fairly involved process. I haven't got any video or documentation for that matter as everything is in my head. You probably should start by buying some books and learning about language models. There is quite a lot information about that on the Internet.

**Comment by Valery813** [comment]
I understand, I will search for information. Thanks

**Comment by palles77** [comment]
I will soon publish these in a separate repo. Stay tuned.

**Comment by Valery813** [comment]
This is excellent

**Comment by Valery813** [comment]
Hello, have you done any transcribing with KALDI? I tried to build the SWE model, a lot of errors in the scripts.

**Comment by palles77** [comment]
No. Its too steep learning curve. My training scripts for Julius allow
state of the art results. They are probably the best training scripts out
there. I will post them soon in a separate repo.

W dniu piątek, 26 lutego 2021 Valery813 <notifications@github.com>
napisał(a):

**Comment by Valery813** [comment]
Hello, what language models will there be?

**Comment by palles77** [comment]
It will be for English. But I am planning to start doing releases for other languages.

**Comment by Valery813** [comment]
Hi, I'm waiting for the release of your model. How to lay out, tell me. ok? I made a model of the Swedish and Danish language for KALDI, the result of transcribing is sad. (%WER = ~5-10%)

**Comment by palles77** [comment]
I am slowly starting to prepare the stuff. Initial repo https://github.com/palles77/htk-cuda. More to follow soon.

**Comment by Valery813** [comment]
Hi, how are you doing with the English model? I tested the aspire and the vosk model alphacep. The results are not very good WER ~ 50%

**Comment by palles77** [comment]
I am progressing this work. I will soon be releasing an upgraded version of Julius. Then I will focus on releasing training procedures for English language. You can always contact me on private email: **silesiaresearch** at **gmail** dot **com**

**Comment by Valery813** [comment]
How are you doing, do you have anything to test ?)))
